PyFlyt/gym_envs/pyflyt_env.py

```diff
@@ -109,8 +109,9 @@ def begin_reset(self, seed=None, options=None, aviary_options=dict()):
         self.info["env_complete"] = False
 
         # init env
-        aviary_options["start_pos"] = np.array([[0.0, 0.0, 1.0]])
+        aviary_options["start_pos"] = np.array([[0.0, 0.0, 10.0]])
```

Comment:
change this to check if "start_pos" in aviary options, else use the default values. If you could, do this for the other 5 lines as well.

Reply:
Not using this file anymore, was messing with it at first as I was planning to use `simple_waypoint.py` env for the fixed wing UAV. Created a new `FWTargets.py` file for the new env instead.